eng/ci/integration-tests.yml

```diff
@@ -0,0 +1,35 @@
+variables:
+  DOTNET_NOLOGO: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+pr:
+  branches:
+    include:
+    - dev
+    - release/4.*
+    - release/inproc6/4.*
+    - release/inproc8/4.*
+
+resources:
+  repositories:
+  - repository: 1es
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc
+      image: 1es-windows-2022 
+      os: windows
+
+    stages:
+    - stage: Test
+      jobs:
+
+      - template: /eng/ci/templates/jobs/initialize-pipeline.yml@self
+      - template: /eng/ci/templates/official/jobs/run-non-e2e-tests.yml@self
+      - template: /eng/ci/templates/official/jobs/run-integration-tests.yml@self
```

Comment:
nit: We have a mix of E2E tests and integration tests, now that we're cleaning up, I would love for the integration tests to be split from E2E

Reply:
Sure - I'll take a look into that.

Reply:
I think I'll leave this for another PR.